webapp/package.json

```diff
@@ -32,7 +32,7 @@
     "@devolutions/icons": "4.0.8",
     "@devolutions/iron-remote-desktop": "0.9.0",
     "@devolutions/iron-remote-desktop-rdp": "^0.6.0",
-    "@devolutions/iron-remote-desktop-vnc": "^0.5.0",
+    "@devolutions/iron-remote-desktop-vnc": "../../ironvnc/web-client/iron-remote-desktop-vnc/dist",
```

Comment:
https://github.com/Devolutions/IronVNC/pull/955 has to be merged and released first.

Reply:
Done